modules/apps/portal-template/portal-template-freemarker/src/main/java/com/liferay/portal/template/freemarker/internal/FreeMarkerManager.java

```diff
@@ -625,9 +629,7 @@ else if (_log.isWarnEnabled()) {
 			}
 		}
 
-		if (sb.index() > 0) {
```

Comment:
@tinatian you removed the index checking because FTL_liferay.ftl should always exist, right? If so, why not just add it directly to the sb. What's point of having it in marcroLibraries first, then test and adding?